virtualhosts/etc/e-smith/templates/etc/opt/rh/rh-php71/php-fpm.d/000-virtualhost.conf/10Base

```diff
@@ -68,6 +69,18 @@ php_value[soap.wsdl_cache_dir]  = /var/opt/rh/rh-$PhpRhVersion/lib/php/wsdlcache
             $OUT .= "php_admin_value[max_execution_time] = $MaxExecTime\n" if ($MaxExecTime ne 'disabled');
             $OUT .= "php_admin_value[max_input_time] =  $MaxExecTime\n" if ($MaxExecTime ne 'disabled');
             $OUT .= "php_admin_value[max_file_uploads] = $MaxFileUploads\n" if ($MaxFileUploads ne 'disabled');
+        } else {
+
+            my $MemoryLimit = $db->get_prop('php','MemoryLimit') || '128';
+            my $UpMaxFileSize = $db->get_prop('php','UploadMaxFilesize') || '2';
+            my $PostMaxSize = $db->get_prop('php','PostMaxSize') || '8';
+            my $MaxExecTime = $db->get_prop('php','MaxExecutionTime') || '0';
+            
+            $OUT .= "php_admin_value[memory_limit] = $MemoryLimit"."M\n" if ($MemoryLimit ne '0');
+            $OUT .= "php_admin_value[upload_max_filesize] = $UpMaxFileSize"."M\n" if ($UpMaxFileSize ne '0');
+            $OUT .= "php_admin_value[post_max_size] = $PostMaxSize"."M\n" if ($PostMaxSize ne '0');
+            $OUT .= "php_admin_value[max_execution_time] = $MaxExecTime\n" if ($MaxExecTime ne 'disabled');
+            $OUT .= "php_admin_value[max_input_time] =  $MaxExecTime\n" if ($MaxExecTime ne 'disabled');
```

Comment:
I feel the `...if ()` can be removed. Isn't the zero (`0`) value implicitly the definition of "disabled"? What are the differences?

Same question for existing lines 65-71: why `...if()` is needed?

Reply:
done, need to test now

Reply:
good, resolved